Participants/Elena Rambla/ceramics.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
+	schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Title</title>
+         </titleStmt>
+         <publicationStmt>
+            <publisher/>
+         </publicationStmt>
+         <sourceDesc>
+            <bibl/>
+         </sourceDesc>
+      </fileDesc>
+   </teiHeader>
+   <text>
+      <body>
+         <entry xml:id="mosaico" xml:lang="sp">
+            <form type="lemma"><orth norm="mosaico">mosaico</orth></form>
+            <gramGrp>
+               <pos norm="NOUN">n.</pos> <pos norm="SING">s.</pos>
```

Comment:
A noun _is_ a part of speech, but singular is a number, not part of speech. Furthermore, I'm not sure that _n.s._ means noun singular. I bet it's _nomen substantivum_, which is really just a noun in Latin. So I would check the introduction of your dictionary just to be sure. 

If it was _noun singular_ then I would suggest `<gram type="pos" norm="NOUN">noun</gram> <gram type="number">singular</gram>`.